airbyte-integrations/connectors/source-google-sheets/.gitignore

```diff
@@ -0,0 +1 @@
+secrets
```

Comment:
is this not handled by the .gitignore at the top of the prooject: https://github.com/airbytehq/airbyte/blob/master/.gitignore#L8